config/gateway/gateway-plugin/gateway-plugin.yaml

```diff
@@ -3,6 +3,12 @@ kind: Service
 metadata:
   name: gateway-plugins
   namespace: aibrix-system
+  labels:
+    app: gateway-plugins
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "8080"
```

Comment:
copied review comment https://github.com/vllm-project/aibrix/pull/1112#discussion_r2155804409
>what's your kubernetes_sd_configs setting? I think prometheus configuration should be under annotation?

Reply:
@Jeffwan You're right.  I guess it was working bcos of service monitor in my env. I missed this. Will fix.